pages/docs/tutorials/kafka/bindings-with-kafka.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Kafka bindings
+description: Learn how to configure Kafka bindings in your AsyncAPI document.
+---
+
```

Comment:
@CynthiaPeter The only thing missing is the `page weight` metadata. Please add one, remembering that the tutorial order places THIS as the fourth and final Kafka tutorial. 😸